src/View/Sidebar.vala

```diff
@@ -2110,6 +2109,7 @@ namespace Marlin.Places {
             var rowref = new Gtk.TreeRowReference (store, path);
             store.@set (iter, Column.SHOW_SPINNER, true);
             store.@set (iter, Column.SHOW_EJECT, false);
+            store.@set (iter, Column.ACTION_ICON, null);
```

Comment:
Not sure why this is necessary.  For this iter, an eject action icon should be appropriate (even if hidden).  It gets set back to an eject icon in `finish_eject_or_unmount ()` anyway.

Reply:
It avoids the eject icon to be displayed aside the spinner, when the eject is finished it sets it back if eject has failed.

Reply:
Ah, OK.  I was just looking at the code - I'll test function asap.